lib/Capabilities.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+/**
+ * SPDX-FileCopyrightText: 2025 Nextcloud GmbH and Nextcloud contributors
+ * SPDX-License-Identifier: AGPL-3.0-or-later
+ */
+
+namespace OCA\Pandoc;
+
+use OCA\Pandoc\AppInfo\Application;
+use OCP\Capabilities\ICapability;
+use OCP\IL10N;
+use OCP\IURLGenerator;
+
+class Capabilities implements ICapability {
+	public function __construct(
+		private IURLGenerator $url,
+		private IL10N $l10n,
+	) {
+	}
+
+	public function getCapabilities(): array {
+		$conversionEndpoint = $this->url->linkToOCSRouteAbsolute('files.conversionapi.convert');
```

Comment:
The endpoint must be relative, since we add the base url on client side, for security reason.